ui/src/flux/components/TimeMachineTables.tsx

```diff
@@ -86,7 +90,10 @@ class TimeMachineTables extends PureComponent<Props, State> {
     } = this.props
 
     return (
-      <div className="time-machine-tables">
+      <div
+        className="time-machine-tables"
+        style={{width: `${width}px`, height: `${height}px`}}
```

Comment:
Previously this component sized itself in pure CSS with some combination of Flexbox and `width: 100%; height: 100%;`. The parent `AutoSizer` messed with this logic (it renders its children in a 0 by 0 pixel `div` with `overflow: visible`). Hence this change.